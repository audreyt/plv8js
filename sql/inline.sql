DO $$ print(NOTICE, 'this', 'is', 'inline', 'code') $$ LANGUAGE plv8;
