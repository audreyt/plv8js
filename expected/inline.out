DO $$ print(NOTICE, 'this', 'is', 'inline', 'code') $$ LANGUAGE plv8;
NOTICE:  this is inline code
